src/scss/_box.scss

```diff
@@ -19,3 +19,9 @@
   background-color: $gray-lighter;
   border-color: transparent;
 }
+
+.box-feature {
```

Comment:
Looks good, add a demo to the Boxes row in the `index.html` so it's obvious it's available.